meson.build

```diff
@@ -1674,7 +1674,7 @@ install_libsystemd_static = static_library(
                         libxz,
                         libzstd,
                         liblz4,
-                        libbpf,
+                        libdl,
```

Comment:
interesting that we didn't need this so far... I mean the "libbasic" meson target needs it, and it compiles a subset of these sources...